src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useRef, useCallback, useMemo, forwardRef } from "react";
+import cx from "classnames";
+import {
+  COLOR_STYLES,
+  COLOR_STYLES_VALUES,
+  CONTENT_COLORS_VALUES,
+  contentColors
+} from "../../../../utils/colors-vars-map";
+import ColorUtils from "../../../../utils/colors-utils";
+import "./ColorPickerItemComponent.scss";
+import Icon from "../../../Icon/Icon";
+import Tooltip from "../../../Tooltip/Tooltip";
+import Clickable from "../../../Clickable/Clickable";
+import { COLOR_SHAPES, COLOR_SHAPES_VALUES } from "../../ColorPickerConstants";
+import { getTestId } from "../../../../tests/test-ids-utils";
+import { ComponentDefaultTestId } from "../../../../tests/constants";
+import { SubIcon, VibeComponent, VibeComponentProps, ElementContent } from "../../../../types";
+import { BASE_SIZES_VALUES } from "../../../../constants";
+
+export interface ColorPickerItemComponentProps extends VibeComponentProps {
+  color: CONTENT_COLORS_VALUES | string;
+  onColorClicked: (color: CONTENT_COLORS_VALUES | string) => void;
+  colorStyle: COLOR_STYLES_VALUES;
+  shouldRenderIndicatorWithoutBackground: boolean;
+  ColorIndicatorIcon: SubIcon;
+  SelectedIndicatorIcon: SubIcon;
+  isSelected: boolean;
+  colorSize: BASE_SIZES_VALUES;
+  tooltipContent: ElementContent;
+  isActive: boolean;
+  colorShape: COLOR_SHAPES_VALUES;
+  ["data-testid"]?: string;
```

Comment:
I think there is no need to add "data-testid" because we declare it in VibeComponentProps